app/models/event.rb

```diff
@@ -4,6 +4,7 @@ class Event < ActiveRecord::Base
   belongs_to :post
   belongs_to :member
   belongs_to :transfer
+  belongs_to :organization
```

Comment:
Is this needed to `Event.where(organization_id: 27)`?

Reply:
nope, it's needed for `event.organization` and the setters (event.organization = organization), at least.

Reply:
Yeah... I meant not literally the field, but 
the duplicated info in the column. if you don't need that kind of `where` you can infer the org from the associated resource

Reply:
Yep, I totally share the same view. Here we're duplicating information that will surely bite us in the shape of data inconsistency.

Reply:
well, in theory ™️ events are read only, an not likely to be moved between orgs.